CHANGES.txt

```diff
@@ -1,6 +1,14 @@
 pyramid_layout changelog
 ========================
 
+Unreleased
+----------
+
+ - Fixed testing for Python 2.7
+ - Added Python compatibility for 3.6, 3.7, 3.8 and 3.9
+ - Removed compatibility for Python 2.6, 3.2 and 3.3. This version and later
+   will require 2.7 or 3.6+
+
```

Comment:
This is good because it will give a stepping stone for users of this library to migrate ultimately to Pyramid 2.0.

In a subsequent PR and release, I think we should drop Python 2.7 support for compatibility with Pyramid 2.0.